nbextensions/bulkImportCell/tabs/xsvGenerator.js

```diff
@@ -202,15 +212,16 @@ define(['StagingServiceClient', 'common/html', 'common/runtime', 'common/ui', 'u
          */
         renderLayout() {
             const state = this.model.getItem('state');
-            let errorMessage = '';
+            let errorMessage;
```

Comment:
Silly question - if `errorMessage` is left undefined, will it wedge an "undefined" or some such thing into teh returned div? I'm guessing not, but just making sure.